JavaScript/Amazon_Price_Tracker/ReadMe.md

```diff
@@ -0,0 +1,11 @@
+
```

Comment:
Thabk you for adding the README, although there are multiple problems with the file.

A detailed list of [warnings](https://github.com/HarshCasper/Rotten-Scripts/runs/2335505295?check_suite_focus=true)

You can either use this list and/or use the [markdown guide](https://github.com/HarshCasper/Rotten-Scripts/blob/master/STYLE_GUIDELINES.md),